Bio/Restriction/Restriction.py

```diff
@@ -155,8 +155,8 @@ def __init__(self, seq, linear=True):
         """FormattedSeq(seq, [linear=True])-> new FormattedSeq.
```

Comment:
We should fix these docstrings which are one line summaries of the arguments and return types counter to PEP257, but that's a separate issue.

Reply:
I changed them (only found two left).